tests/CMakeLists.txt

```diff
@@ -1447,6 +1242,7 @@ experimental_tests(cgalpngtest_cube-with-hole
                    astdumptest_assert-expression-fail1-test
                    astdumptest_assert-expression-fail2-test
                    astdumptest_assert-expression-fail3-test
+		   echotest_issue1851-each-fail-on-scalar
```

Comment:
This appears to be an artifact caused by https://github.com/openscad/openscad/pull/2596/commits/9f6ba917f3e11115a1574b9d3dd1bfb44bc74147 (adding blank line after this line)  https://github.com/openscad/openscad/commit/83e1b6c0d68de2a7aaaaf3790ee9647b224dd581 (removing this line) and https://github.com/openscad/openscad/pull/2596/commits/f09849bf5aa0511822bf1d43daa5848d8ccfc138 (updating to current master)
```suggestion
```